credentials/xds/xds_client_test.go

```diff
@@ -219,11 +220,13 @@ func newTestContextWithHandshakeInfo(parent context.Context, root, identity cert
 	// Creating the HandshakeInfo and adding it to the attributes is very
 	// similar to what the CDS balancer would do when it intercepts calls to
 	// NewSubConn().
-	info := xdsinternal.NewHandshakeInfo(root, identity)
+	var sm []matcher.StringMatcher
```

Comment:
Nit/optional: `sms` to pluralize string matcher.

Reply:
Fair. Switched.